cpu/nrf5x_common/clock.c

```diff
@@ -28,6 +30,8 @@
 #error "Clock init: CLOCK_LFCLK is not defined by your board!"
 #endif
 
+static volatile unsigned _hfxo_requests = 0;
```

Comment:
Why is this `volatile`?

Reply:
unintentional leftover from the first prototype version. Now that I put reading/writing that var in critical sections, there is no need for it anymore... fixed.